application/src/test/java/org/thingsboard/server/controller/AlarmControllerTest.java

```diff
@@ -796,4 +800,101 @@ public void testCreateAlarmWithOtherCustomerAsAssignee() throws Exception {
         doPost("/api/alarm", alarm).andExpect(status().isForbidden());
     }
 
+    @Test
+    public void testGetAlarmTypes() throws Exception {
```

Comment:
The action you test is 'saveAlarmTypes'. The way HOW you test the desired state is the `getAlarmTypes`. The 'getAlarm' is present in all test cases. So the method name is confusing. Please, rename it like 'testSaveAlarmTypes' or similar meaning